README.md

```diff
@@ -372,6 +372,15 @@ If you have received the package and only want to build and run the docker conta
         concent-signing-service:latest
     ```
 
+### Generate Signing Service keys
+To generate public and private keys for signing-service use the `generate-ecc-key-pair.sh` script.
+```bash
+docker run                                  \
+    --interactive                           \
+    --tty                                   \
+    --entrypoint generate-ecc-key-pair.sh   \
+    concent-signing-service
+```
```

Comment:
That's not the purpose of `--entrypoint`. It would be better to set `--entrypoint /bin/bash` and run the command as usual. This makes it clear that you simply want to override the default entrypoint.